File: cmt-api/1. BO/CMT.BO/Admin/ParentValueAttribute.cs
using System.Runtime.Serialization;

namespace CMT.BO.Admin
{
    [DataContract(Name = "ParentValueAttribute")]
    public class ParentValueAttribute
    {
        [DataMember]
        public string ParentId { get; set; }
        [DataMember]
        public string ElementName { get; set; }
        [DataMember]
        public string AttributeValue { get; set; }
        [DataMember]
        public int ElementType { get; set; }
        [DataMember]
        public string Color { get; set; }
    }

    public enum AttributeElementType
    {
        BU = 8,
        SUBBU = 16,
        PORTFOLIO = 32,
        INDICATION = 64
    }
}

File: cmt-ui/src/app/shared/parent-value-attribute.ts
export class ParentValueAttribute {
	ParentId: string;
	ElementName: string;
	AttributeValue: string;
	ElementType: AttributeElementType;
	Color: string;
}

export enum AttributeElementType {
	BU = 8,
	SUBBU = 16,
	PORTFOLIO = 32,
	INDICATION = 64
}
